src/index.html

```diff
@@ -11,12 +11,111 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header-nav">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="MOYO Logo"
+        />
+      </a>
+
+      <nav class="nav-bar">
+        <ul class="nav-list">
```

Comment:
Learn to write classes right away, use the best example for your classes
<img width="687" alt="Знімок екрана 2024-08-13 о 14 52 09" src="https://github.com/user-attachments/assets/0d2c65ca-abc6-413f-8be8-9666637fac49">

```suggestion
    <header class="header">
      <a
        href="index.html"
        class="logo"
      >
        <img
          src="images/logo.png"
          alt="MOYO Logo"
        />
      </a>

      <nav class="nav">
        <ul class="nav__list">
```